pkg/fleetautoscalers/controller.go

```diff
@@ -293,3 +336,68 @@ func (c *Controller) updateStatusUnableToScale(ctx context.Context, fas *autosca
 
 	return nil
 }
+
+// createFasThread creates a FleetAutoScaler sync routine
+func (c *Controller) createFasThread(ctx context.Context, fas *autoscalingv1.FleetAutoscaler) error {
+	key, err := cache.MetaNamespaceKeyFunc(fas)
+	if err != nil {
+		return err
+	}
+	ticker := c.clock.NewTicker(time.Duration(fas.Spec.Sync.FixedInterval.Seconds) * time.Second)
+	c.fasThreads[key] = fasThread{
```

Comment:
Does `fasThreads` need to be protected by a mutex to prevent simultaneous writes?

Reply:
I'm not sure. Will one FAS be synced twice simultaneously? If so we may need a mutex.

Reply:
Use sync.map instead of map